Pipeline/controllers/BayesPairingController.py

```diff
@@ -0,0 +1,44 @@
+from services.BayesPairing import bayespairing_string, bayespairing_file, get_graphs_per_module
+import tempfile
+
+
+class BayesPairing:
+
+    @staticmethod
+    def bayespairing_string(arguments):
+        '''
+        Processes BayesPairing input before calling its service.
+
+        :param arguments: a dictionary containing all arguments to BayesPairing
+        :returns: the response from the BayesPairing service
+        '''
+        return bayespairing_string(arguments)
```

Comment:
is this recursive? I don't understand what's happening here (unless it's just a temporary thing because it's still incomplete... if that's the case then ignore what i said hihi)

edit: i'm literally dumb it legit says it's the service function call... it all makes sense now

Reply:
No, just bad naming

Reply:
Will address bad naming in a follow-up later so we can get this into master ASAP.